docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.angular.mdx

```diff
@@ -9,6 +9,10 @@ import AWSExports from './shared/add-aws-exports-type.mdx';
 
 <AWSExports />
 
+### Server-side Rendering (SSR)
```

Comment:
Small request, can you please do a minor cleanup of this page by aligning the heading sizes of this file to match the vue and react pages?